app/models/metric.rb

```diff
@@ -3,7 +3,22 @@ class Metric < ApplicationRecord
 
   include Metric::Common
 
-  def self.reindex_table_name
-    "metrics_#{Time.now.utc.hour + 1}"
```

Comment:
we lost the `+1` here
this may effect reindexing which actually wants to reindex the next hour